README.md

```diff
@@ -1,82 +1,200 @@
+1.  VPN (Optional)
+
+
+Finally [deploying](https://cloud.google.com/migrate/compute-engine/docs/4.5/how-to/configure-manager/configuring-on-gcp) Migrate for Compute Engine Manager via GCP [Marketplace](https://console.cloud.google.com/marketplace/details/click-to-deploy-images/velostrata?_ga=2.230596124.-1830265044.1554384916&_gac=1.75634663.1564563946.CL6bne_m3uMCFYYkGwodLkkPoQ).
+
+
+## Getting Started
+
+
+#### Configuring Migrate for Compute Engine Landing Zone with CFT
+
+
+### Tools
+
+**Install Cloud SDK** \
+The Google Cloud SDK is used to interact with your GCP resources. [Installation instructions](https://cloud.google.com/sdk/downloads) for
+
+multiple platforms are available online.
+
+**Install Terraform** \
+Terraform is used to automate the manipulation of cloud infrastructure. Its [installation instructions](https://www.terraform.io/intro/getting-started/install.html) are also available online.
+
+**Authenticate gcloud** \
+Prior to running this, ensure you have authenticated your gcloud client by running the following command:
+
+
+
+```
+gcloud auth application-default login
+```
+
+
+
+### Deployment
+
+Download the repository:
+
+
+```
+git clone https://github.com/terraform-google-modules/terraform-google-migrate.git
+```
+
+
+File structure
+
+The project has the following folders and files:
 
-```hcl
-module "migrate" {
-  source  = "terraform-google-modules/migrate/google"
-  version = "~> 0.1"
 
-  project_id  = "<PROJECT ID>"
-  bucket_name = "gcs-test-bucket"
-}
 ```
 
-Functional examples are included in the
-[examples](./examples/) directory.
+/: root folder
+/examples: Examples for doing single and multi project deployments
+/modules: Modules for core, single and multi projects
+/helpers: Optional helper scripts for ease of use
+/main.tf: TODO
+/variables.tf:TODO
+/output.tf: TODO
+/readme.md: this file
 
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-## Inputs
+```
+**Single project deployment:**
 
-| Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-----:|:-----:|
-| bucket\_name | The name of the bucket to create | string | n/a | yes |
-| project\_id | The project ID to deploy to | string | n/a | yes |
+We will start with deploying our **single** project:
 
-## Outputs
 
-| Name | Description |
-|------|-------------|
-| bucket\_name |  |
+```
+cd examples/single_deployment
+```
 
-<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 
-## Requirements
+Provide this variables in terraform.tfvars:
 
-These sections describe requirements for using this module.
 
-### Software
+```
+organization_id = "GCP ORGANIZATION ID"
+billing_account = "GCP BILLING ID"
+credentials_path = "SA KEY USED TO PROVISION RESOURCES"
+subnet_01_ip = "GCP VPC SUBNET IP"
+subnet_02_ip = "GCP VPC SUBNET IP"
+subnet_03_ip = "GCP VPC SUBNET IP"
+secondary_s01_range = "SECONDARY IP VPC SUBNET RANGE"
```

Comment:
Do we really need secondary ranges? They're mostly not applicable to GCE.

Reply:
Removed